src/js/stage.js

```diff
@@ -213,8 +213,11 @@ define(["d3", "palette", "transitions/default"], function (d3, palette, defaults
           .data(function (d) {
             return d.labels;
           });
-        labels.enter().append("text");
         transitions.labels(labels, labelAttributer);
+        labels.enter()
+          .append("text")
+          .call(labelAttributer)
+          .call(styliseur);
```

Comment:
Stage should not have dependency to `styliseur`. It should be moved to `default/transitions` if it is really needed. In fact transitions for labels looks as follows:
```js
      labels
        .transition()
        .delay(150)
        .duration(900)
        .call(attributer)
        .call(styliseur);
```
Please make it consistent.